uPortal-webapp/src/main/resources/properties/contexts/applicationContext.xml

```diff
@@ -124,11 +124,27 @@
         </constructor-arg>
     </bean>
 
+    <!-- And JGROUPS DDL used by PostgresSQL Server -->
+    <bean id="jgroupsPostgreSQLInitSql" class="java.lang.String">
+        <constructor-arg>
+            <value>
+                CREATE TABLE IF NOT EXISTS JGROUPSPING (
```

Comment:
:question: would adding `IF NOT EXISTS` to the default query break a different database?

Reply:
The SQL is specific to Postgres via dialect mapping in `jgroupsInitSqlMap` below.

Reply:
Right, could it be added to the default query, to avoid a special condition for Postgres?

Reply:
No. There are databases that do not recognize `IF NOT EXISTS`, unfortunately. I believe Oracle is one.

Also, the Postgres code is custom for the ping_data column type (?).

Reply:
:white_check_mark: